package.json

```diff
@@ -39,7 +39,7 @@
     "typescript": "^4.4.4"
   },
   "dependencies": {
-    "@tracer-protocol/perpetual-pools-contracts": "^1.2.3",
```

Comment:
wouldn't expect this change, is this deliberate?

Reply:
yeah I updated with getFee and getLeverage

Reply:
but will have to wait for redeploy fook

Reply:
have bumped to 1.2.5 which is latest version